packages/babel-plugin-proposal-pipeline-operator/test/fixtures/hack-hash/basic/exec.js

```diff
@@ -0,0 +1,3 @@
+const result = 5 |> # + 1 |> # + #;
+
+expect(result).toBe(12);
```

Comment:
Can you also add `input`/`output` fixtures for all these new exec tests?
You can copy the `exec.js` files to `input.js` in the same folders, and just remove the `exec` assertions.

Reply:
I had thought that `exec.js` tests were preferred to `input`/`output.json`? That’s what [CONTRIBUTING.md](https://github.com/babel/babel/blob/main/CONTRIBUTING.md) says, at least: 

> Generally, there are two kinds of tests for plugins
> * The first is a simple test of the input and output produced by running Babel on some code. We do this by creating an input.js file and an output.js file.  
> * The second and preferred type is a test that actually evaluates the produced code and asserts that certain properties are true or false. We do this by creating an exec.js file.

Reply:
Oh, wait, so we want *both* `exec.js` *and* `input`/`output.js` files. I can do that.

Reply:
Fixed in 2bd98d2b1bc09d590fb7cca04159d0f31a33bfb0.